specs/beacon-chain.md

```diff
@@ -209,6 +211,12 @@ The `CrystallizedState` has the following fields:
     'deposits_penalized_in_period': ['uint32'],
     # Hash chain of validator set changes (for light clients to easily track deltas)
     'validator_set_delta_hash_chain': 'hash32'
+    # Genesis time
+    'genesis_time': 'hash32',
+    # PoW chain reference
+    'known_pow_receipt_root': 'hash32',
+    'candidate_pow_receipt_root': 'hash32',
+    'candidate_pow_receipt_root_votes': 'uint64',
```

Comment:
This can be a smaller int type as the value should never exceed `POW_HASH_VOTING_PERIOD`